components/item-detail/item-detail.tsx

```
@@ -37,10 +40,29 @@ export interface ItemDetailProps {
+		url: `${siteConfig.url}/${locale}/items/${meta.slug}`,
+		category: categoryName,
+		sourceUrl: meta.source_url,
+		brandName: siteConfig.brandName
+	});
+
 	return (
 		<div className="min-h-screen bg-gradient-to-br from-gray-50 via-white to-blue-50/30 dark:from-gray-950 dark:via-gray-900 dark:to-gray-800 text-gray-800 dark:text-white relative overflow-hidden">
+			{/* Product Schema JSON-LD */}
+			<script
+				type="application/ld+json"
+				dangerouslySetInnerHTML={{ __html: JSON.stringify(productSchema) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)